components/multiselect/custom-value.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Custom Value
+page_title: MultiSelect - Custom Value
+description: Custom values and user input in the MultiColumnComboBox for Blazor.
+slug: multiselect-custom-value
+tags: telerik,blazor,multiselect,custom,value,input
+published: True
+position: 16
+---
+
+# MultiSelect Custom Values
+
+The MultiSelect component allows the user to type in their own value that is not a part of the predefined set of options that the developer provided.
+
+The text entered by the user can still go into the field the combo box is bound to through two-way binding.
+
+To enable custom user input, set the `AllowCustom` parameter to `true`. When the user types a custom value, it will appear as the first item in the list with the label: `Use"typed value"`. Refer to the example below to see it in action.
```

Comment:
```suggestion
To enable custom user input, set the `AllowCustom` parameter to `true`. When the user types a custom value, it appears as the first item in the list with the label: `Use"typed value"`. See the example below for details.
```

Reply:
Here we can add information how the user can add the custom value. My understanding for the feature is that the user must click the `Use"typed value"` in the popup and the custom value is added to the component `Value`. Before that, the MultiSelect does not actually have a custom selected value, right?